src/derivers/bip32.ts

```diff
@@ -1,13 +1,13 @@
-import crypto from 'crypto';
```

Comment:
Potential question for @rekmarks whether we prefer to use `crypto` where possible?

Reply:
This is a good question; I'm not completely sure. My hunch is that we'd prefer `crypto` when on Node, but it seems like `@scure` / `@noble` has avoided using `crypto`, and I wonder why that is. It seems unlikely that anything else would have better performance.